fixtures/mock-oauth2-provider/package.json

```diff
@@ -54,5 +49,11 @@
     "lodash": "^4.17.15",
     "passport-http": "^0.3.0",
     "passport-oauth2": "^1.5.0"
+  },
+  "devDependencies": {
+    "@loopback/build": "^5.3.1",
+    "@loopback/eslint-config": "^6.0.6",
+    "tslib": "^1.11.2",
```

Comment:
This needs to be in `dependencies`.

Reply:
@raymondfeng all loopback packages refer only tslib 1.12.0 as of now

Reply:
agreed